src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const thumbs = document.querySelector('.gallery__list');
+
+thumbs.addEventListener('click', (ev) => {
+  ev.preventDefault();
+
+  const mainImage = document.querySelector('#largeImg');
+  const src = ev.target.closest('.list-item__link').href;
```

Comment:
Ensure that the clicked element or its ancestors have the class 'list-item__link'. If the click event occurs on a child element that does not have this class, `closest` will return `null`, leading to an error when accessing `href`. Consider adding a check to ensure `ev.target.closest('.list-item__link')` is not null before accessing `href`.